CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Changed from version 0.12.0 to master
 
+- Added advertised endpoint to coordinators and active fo
```

Comment:
`active fo` ??

Reply:
active failover?

Reply:
"Added advertised endpoint to cluster & activefailover deployments."